.github/workflows/test.yml

```diff
@@ -4,35 +4,11 @@ on: [push]
 
 jobs:
   xcode_tests:
-    name: ${{ matrix.platform }} Tests (Xcode ${{ matrix.xcode }})
+    name: Tests
     runs-on: macos-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        xcode: ["11.5"]
-        platform: ["iOS"]
```

Comment:
I suppose we could more platforms for now the package only supports iOS so this was removed